ofrak_core/CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Remove `Data` attribute to eliminate duplicative indexing in `ResourceService`, resulting in performance improvement for all workflows ([#589](https://github.com/redballoonsecurity/ofrak/pull/589))
 
 ### Changed
+- Replace `ofrak.injector.discover` with `ofrak.discover` in OFRAK tests ([#592](https://github.com/redballoonsecurity/ofrak/pull/592))
```

Comment:
Don't think we need to list this as a change. We should rev the rc version for this thought and have an entry about updating the OFRAK documentation. You'll need to change rcX both here and in the `ofrak_core/setup.py`.